### PR TITLE
Switch kubevirt metric for AllocatedMB field.

### PR DESCRIPTION
**Pull request by andrewd-zededa**
kubevirt_vmi_memory_domain_bytes_total doesn't exist. Add comments with link to docs for easier lookup later.

### CODE REVIEW

**Review by zedi-pramodh — APPROVED** [review]
LGTM

**Comment by zedi-pramodh** [comment]
But will wait for Naiming review before merging

**Review by naiming-zededa — APPROVED** [review]
The kubevirt metrics '_total' was removed in this commit:
https://github.com/kubevirt/kubevirt/pull/9821/files

LGTM

**Pull request merged**
